media-libs/raylib/raylib-5.0.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A simple and easy-to-use library to enjoy videogames programming"
+DESCRIPTION="Raylib is highly inspired by Borland BGI graphics \
+lib and by XNA framework and it's especially well suited for \
+prototyping, tooling, graphical applications, embedded systems and education."
+HOMEPAGE="https://github.com/raysan5/raylib"
+COPYRIGHT="2013-2024 Ramon Santamaria"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://github.com/raysan5/raylib/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="98f049b9ea2a9c40a14e4e543eeea1a7ec3090ebdcd329c4ca2cf98bc9793482"
+SOURCE_FILENAME="$portVersion.tar.gz"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	raylib = $portVersion
+	devel:libraylib = $portVersion
```

Comment:
these are both missing `$secondaryArchSuffix`

Reply:
Additionally, this is missing `lib:libraylib$secondaryArchSuffix = $portVersion`. The `devel:` entry should be moved to a `_devel` subpackage. Please see existing library packages as examples how to do this.

Reply:
by default it configures a static library. there is an option to enable shared libs (_BUILD_SHARED_LIBS_). i've applied that option and tried to restructure per suggestion.